packages/gatsby-remark-prismjs/src/highlight-code.js

```diff
@@ -16,30 +16,43 @@ module.exports = (
   diffLanguage = null
 ) => {
   // (Try to) load languages on demand.
-  if (!Prism.languages[language]) {
```

Comment:
was removal of this check intended?

Reply:
To answer my own question (somewhat):

`loadPrismLanguage` have rather similar check in https://github.com/gatsbyjs/gatsby/blob/aa100f77bf4182ac3b4aa687c29689199cd2513a/packages/gatsby-remark-prismjs/src/load-prism-language.js#L20-L30

I don't think this breaks anything, but there is slight difference tho, this adds bit of overhead of calling `getBaseLanguageName` every time - maybe instead of removing it completely can we move this check and bail early to `loadPrismLanguage` function (before calling `getBaseLanguageName`) to avoid unnecessary work (likely will not be felt on smaller sites, but everything at scale does :( )

Reply:
I see. I restore the checking logic and add some comments for diffLanguage.